forest/cli/src/cli/snapshot_fetch.rs

```diff
@@ -407,4 +487,28 @@ mod test {
             assert!(replace_extension_url(case.0.try_into().unwrap(), case.1).is_err())
         });
     }
+
+    #[test]
+    fn download_with_aria2_test_wrong_checksum() -> Result<()> {
```

Comment:
Do these tests work if `aria2` isn't installed?

Reply:
It will fail, would u suggest make it pass w/o aria2 installed?

Reply:
Yeah, I'd like for `aria2` to be an optional dependency for developers.

Reply:
Fixed